### ISSUE TITLE
Adding snoack to the Jinja DevTeam

### ISSUE THREAD

**Issue by untitaker** [pallets/meta#14]
Hello @snoack,

I added you to the new organization "pallets" that contains Flask, Werkzeug and Jinja. Since you've been rather active on Jinja's bugtracker, you currently have commit access to just that repo (if you accept the invite).

Our policies are work-in-progress, the tracking issue is https://github.com/pallets/meta/issues/10 (and generally that issuetracker). Especially the first point about backwards compat is non-negotiable, so use your new privileges wisely :)

If you want access to other repos, please let me know.

--

I'm going to close this issue, I only posted it here for transparency.

**Comment by davidism** [comment]
@snoack be sure to add yourself as a watcher to at least pallets/jinja and pallets/meta so you can keep up with issues and discussion.

**Comment by snoack** [comment]
Thanks for the invitation! I accepted it.

@untitaker I read through #10. And yeah, I'm well aware of our policy in regard of backwards compatibility. However, I was wondering what exactly my responsibilities as a committer are, and when/how it's appropriate to push a change?

@davidism Yes, I'm already watching `jinja` for a while, I just setup notifications for `meta` as well.

**Comment by untitaker** [comment]
@snoack Technically you have no responsibilities, this is voluntary open source. But if you do want to see a PR merged into master, please do that while respecting #10.

**Comment by jeffwidman** [comment]
> when/how it's appropriate to push a change?

I'd say use your judgment and when in doubt, have a bias towards action rather than apathy... action can mean rejecting a PR/issue too--they're easy enough to re-open if someone disagrees.

Normally when a PR comes in, I might comment immediately, then wait a few days before merging, even if I'm confident the change should be implemented, just to give other maintainers a chance to weigh in. 

If I'm the one changing the code, and I think it might be controversial, I submit a PR and solicit feedback. If it's a trivial change like a typo, then I commit it without waiting on a PR. 

Also, depends on whether it's internal restructuring vs changing the external API vs changing docs. 

The external API changes are the ones I'm most cautious about. For example, I'd be hesitant to say 'yes' to a new custom filter unless I'd gotten a thumbs up from another maintainer... just because it's so difficult to remove those later. I'd just bump the PR with a comment asking for input. Also, if the submitter forgot to add tests for the new features, ask for them. 

Docs changes are generally straightforward to review... feel free to reject them if they don't actually improve things, that happens more than you might think. 

For internal cleanup, if a PR has sat there for a bit and no one else is chiming in, then if you think it's good, I'd say go ahead and merge it. You've got a few outstanding PRs like that, if you still think they're good to merge, then pull the trigger.

Others on the team may handle things differently, but that's what I've been doing. It seems to strike a decent balance between not acting rashly or stepping on others toes, but also making sure things keep moving forward and PRs/issues don't sit in limbo forever.

**Comment by untitaker** [comment]
Also FWIW, if a PR includes a refactor of some code, be aware that passing tests don't indicate anything, because the test coverage is really bad at some places. Always demand additional tests when somebody does that.

**Comment by snoack** [comment]
Thanks, that is helpful! I will try doing my best.

Though, I'm currently quite busy with other stuff. But I probably will have more time again for `jinja` soon.

**Comment by jeffwidman** [comment]
@snoack I noticed you haven't been active for a while, there were a couple of your old PR's that others asked you questions about and you never chimed in...

Are you still interested in being involved or has life taken you in another direction?
